Zeeguu Reader/Pods/SwiftyJSON/README.md

```diff
@@ -0,0 +1,376 @@
+#SwiftyJSON [中文介绍](http://tangplin.github.io/swiftyjson/)
+
+[![Travis CI](https://travis-ci.org/SwiftyJSON/SwiftyJSON.svg?branch=master)](https://travis-ci.org/SwiftyJSON/SwiftyJSON)
+
+SwiftyJSON makes it easy to deal with JSON data in Swift.
```

Comment:
Oh man.
And XCode 7.3 requires OS X 10.11!
You've given me work!!

On Tue, Mar 29, 2016 at 10:24 AM, Mircea Filip Lungu <mircea.lungu@gmail.com
> wrote:

> I had a look at the documentation that you sent.
>
> I was thinking about the "benefits of ignoring the pods directory" when I
> asked the question.
> But I see that there are benefits in also checking in the pods...
> especially in times like these:
>
>   http://www.haneycodes.net/npm-left-pad-have-we-forgotten-how-to-program/
>
> M.
>
> On Tue, Mar 29, 2016 at 10:12 AM, JorritO <notifications@github.com>
> wrote:
>
>> In Zeeguu Reader/Pods/SwiftyJSON/README.md
>> <https://github.com/mircealungu/Zeeguu-Reader-iOS/pull/5#discussion_r57685893>
>> :
>>
>> > @@ -0,0 +1,376 @@
>> > +#SwiftyJSON [中文介绍](http://tangplin.github.io/swiftyjson/)
>> > +
>> > +[![Travis CI](https://travis-ci.org/SwiftyJSON/SwiftyJSON.svg?branch=master)](https://travis-ci.org/SwiftyJSON/SwiftyJSON)
>> > +
>> > +SwiftyJSON makes it easy to deal with JSON data in Swift.
>>
>> No, cocoapods isn't part of Xcode and will therefore not be run at build
>> time. When you do pod install, it will download all dependencies (if
>> needed, won't update them to newer versions, see pod install vs pod
>> update <https://guides.cocoapods.org/using/pod-install-vs-update.html>). pod
>> install creates an Xcode workspace that contains the project (Zeeguu
>> Reader in this case) and a Pods project, that contains all pods. That
>> project and the workspace are configured in such a way that every pod and
>> the project are built and put together, but it won't run pod install or
>> something.
>>
> Aha.
>
>
>
>> Now on why that readme is there. I chose to commit the Pods folder (see
>> also the cocoapods documentation
>> <https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control>).
>> That readme is there, because it is downloaded by cocoapods and put into
>> the Pods folder.
>>
>> —
>> You are receiving this because you commented.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mircealungu/Zeeguu-Reader-iOS/pull/5/files/de5a64f5215714b910a03c681f0a56c424769205#r57685893>
>>
>
>